src/charm.py

```diff
@@ -254,6 +265,13 @@ def __init__(self, *args):
             self._on_recv_ca_cert_removed,
         )
 
+        observe(
+            self.forward_auth.on.forward_auth_config_changed, self._on_forward_auth_config_changed
```

Comment:
I agree that it would look ambiguous if we named a custom event similarly to a built-in one.
Changed to `auth_config_changed` and `auth_config_removed`.